src/test/resources/network/brightspots/rcv/test_data/test_set_1_exhaust_at_overvote/test_set_1_exhaust_at_overvote_cvr.json

```diff
@@ -799,45 +799,44 @@
   "Election" : [ {
     "@id" : "election-001",
     "@type" : "CVR.Election",
+    "Candidate" : [
```

Comment:
Here you see a very simple example of top-level "static" Candidate objects.  One could attach more Candidate info here e.g. a PartyId.